Xamarin.Essentials/Email/Email.win32.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Threading.Tasks;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Email
+    {
+        internal static bool IsComposeSupported =>
+            throw ExceptionUtils.NotSupportedOrImplementedException;
+
+        static Task PlatformComposeAsync(EmailMessage message) =>
+            throw ExceptionUtils.NotSupportedOrImplementedException;
+    }
+
+#if NETSTANDARD1_0 || NETSTANDARD2_0
```

Comment:
Not sure what this #if is for? This file should not be included for those defines.

Reply:
Sorry, that file is a placeholder copy-pasted. I'll add in the Email implementation today.

Reply:
Replaced with actual Win32 implementation